dotnet/src/SemanticKernel.Test/AI/OpenAI/Services/AzureOpenAIConfigTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel.AI.OpenAI.Services;
```

Comment:
per previous comment - These files and other in this test project are bad merge results, I believe you just need to delete this and BackendConfigTests, OpenAIConfigTests, and CodeBlockTests.

Reply:
@awharrison-28 Let me to do clean pull from main to a branch and copy all the new files fresh to ensure that there is not file overlap another bad merge.